### PR TITLE
{AKS} Set disable local accounts to true when creating an automatic cluster

### PR DESCRIPTION
**Pull request by xmzhao0822**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### Related command
az aks create -g xmrg -n xmmc --sku automatic

### General Guidelines

- [x] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [x] Have you run `python scripts/ci/test_index.py -q` locally? (`pip install wheel==0.30.0` required)
- [x] My extension version conforms to the [Extension version schema](https://github.com/Azure/azure-cli/blob/release/doc/extensions/versioning_guidelines.md)

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your pull request is merged into main branch, a new pull request will be created to update `src/index.json` automatically.  
You only need to update the version information in file setup.py and historical information in file HISTORY.rst in your PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
AKS

File: src/aks-preview/azext_aks_preview/managed_cluster_decorator.py [hunk 1]
```diff
@@ -336,6 +336,81 @@ def get_enable_msi_auth_for_monitoring(self) -> Union[bool, None]:
...
+    def get_enable_local_accounts(self) -> bool:
+        """Obtain the value of enable_local_accounts.
+
+        This function will verify the parameter by default. If both disable_local_accounts and enable_local_accounts
+        are specified, raise a MutuallyExclusiveArgumentError.
+
+        :return: bool
+        """
+        return self._get_enable_local_accounts(enable_validation=True)
+
```
Comment (FumingZhang) [inline src/aks-preview/azext_aks_preview/managed_cluster_decorator.py:413]:
```suggestion
```

File: src/aks-preview/HISTORY.rst [hunk 2]
```diff
@@ -13,6 +13,10 @@ Pending
 +++++++
 * Add option `--enable-advanced-network-observability`, `--disable-advanced-network-observability` to `az aks create/update`
 
+3.0.0b13
+++++++++
+* Set disable local accounts to true when creating an automatic cluster
+
```
Comment (FumingZhang) [inline src/aks-preview/HISTORY.rst:19]:
```suggestion

3.0.0b13
++++++++
* Set disable local accounts to true when creating an automatic cluster
* Add option `--enable-advanced-network-observability`, `--disable-advanced-network-observability` to `az aks create/update`
```

File: src/aks-preview/azext_aks_preview/managed_cluster_decorator.py (hunk 1 above)
Comment (FumingZhang) [reply src/aks-preview/azext_aks_preview/managed_cluster_decorator.py:413]: actually you don't need to write down the same thing again

**Comment by FumingZhang** [comment]
The change looks good, waiting for feature owner to provide a proof of passed [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=93041978&view=results)

**Comment by xmzhao0822** [comment]
> The change looks good, waiting for feature owner to provide a proof of passed [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=93041978&view=results)

Do I need to push the recordings change at here?

**Comment by FumingZhang** [comment]
> > The change looks good, waiting for feature owner to provide a proof of passed [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=93041978&view=results)
> 
> Do I need to push the live passed recordings change at here?

the recording file is not that important, it would be great if the live test pipeline could pass, I queued another one in [eastus](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=93046175&view=results)

**Comment by xmzhao0822** [comment]
> > > The change looks good, waiting for feature owner to provide a proof of passed [live test](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=93041978&view=results)
> > 
> > 
> > Do I need to push the live passed recordings change at here?
> 
> the recording file is not that important, it would be great if the live test pipeline could pass, I queued another one in [eastus](https://dev.azure.com/msazure/CloudNativeCompute/_build/results?buildId=93046175&view=results)

The live test pipeline in the EastUS has passed :)

**Review by FumingZhang — APPROVED** [review]

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**